### ISSUE TITLE
Unable to resolve module `ReactNativeART`

### ISSUE THREAD

**Issue by moksamedia** [blackuy/react-native-twilio-video-webrtc#59]
Any suggestions on resolving this issue? Thanks.

Full error: Unable to resolve module `ReactNativeART` from `/Users/cantgetnosleep/development/toptal/TulsaApp/bodak-frontend/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module does not exist in the module map

- **Node.js version**: v7.10.1
- **React Native version**: 0.47.2
- **React Native platform + platform version**: iOS 9.0

### react-native-twilio-video-webrtc
**Version**: 1.0.2 from yarn

I tried adding subspecs to the podfile to resolve this. Also tried numerous rounds of cleaning and rebuilding both node_modules and the pod project.

Podfile:
`platform :ios, '9.0'

target 'SmileMore' do

  pod 'Yoga', path: '../node_modules/react-native/ReactCommon/yoga/Yoga.podspec'
  pod 'React', path: '../node_modules/react-native', :subspecs => [
      "Core",
      "ART",
      "RCTActionSheet",
      "RCTAnimation",
      "RCTCameraRoll",
      "RCTImage",
      "RCTGeolocation",
      "RCTNetwork",
      "RCTText",
      "RCTVibration",
      "RCTWebSocket",
      "DevSupport",
      "BatchedBridge"
     ]
  pod 'react-native-twilio-video-webrtc', path: '../node_modules/react-native-twilio-video-webrtc'

end`

**Comment by bilby91** [comment]
Hey, not sure what could be the source of the problem. What version of react are you using ?

I'm closing this issue since it's not related to `react-native-twilio-video-webrtc`

**Comment by moksamedia** [comment]
React version:     "react": "16.0.0-alpha.12",

"I'm closing this issue since it's not related to react-native-twilio-video-webrtc" -- I'm not entirely sure that's true because I didn't have this problem until I tried to integrate react-native-twilio-video-webrtc.

**Comment by bilby91** [comment]
Hey,

Good, let's reopen the ticket and try to figure out what is going on.

So, the problem only happens when you include the dependency ? Is that the only change the triggers the problem ? Would it be possible to upgrade react-native to the latest version and verify the problem is still present ?

**Comment by moksamedia** [comment]
Sure. 0.49? I can try that. I do appreciate you assistance, by the way. 

Should I need to add the subspecs to the podfile to get this to work?

**Comment by moksamedia** [comment]
So.... It's resolved itself. Not sure exactly what was going on. Probably some weird state with the cache or packager. 

I decided to revert back to the pre-react-native-twilio-video-webrtc state (which I had committed just in case this turned into a "thing"--sometimes we do learn from experience). And after clearing caches and what not, I was still getting the same error.

I shut everything down and restarted the computer and the pre-module install state worked again. Re-installed the module and I'm in business.

Whew. No idea what was going on but at least it appears to be working.
